Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -60,23 +54,32 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _hoverWrapper = hoverWrapper;
         }
 
-        private void Gripper_KeyDown(object sender, KeyRoutedEventArgs e)
+        private void CreateGripperDisplay()
         {
-            var gripper = sender as GridSplitterGripper;
-
-            if (gripper == null)
+            if (_gripperDisplay == null)
```

Comment:
No need for this check as _gripperDisplay will always be null if this function is reached